cmd/lncli/commands.go

```diff
@@ -1135,10 +1135,12 @@ var createCommand = cli.Command{
 			Name: "multi_backup",
 			Usage: "a hex encoded multi-channel backup obtained " +
 				"from exportchanbackup",
+			TakesFile: true,
```

Comment:
Could split the PR into two commits, one bumping the version and adding these flags, the other one for the `fish-completion`.